### DISCUSSION TITLE
How can I get other versions in json format

### DISCUSSION THREAD

**Discussion by foozaweb** [eliranwong/UniqueBible#1388] [Q&A]
I'm not a Python developer, but I really would love to have the Bible in the following translation
AMP, NIV, TPT, MSG, ESV, NASB & SVV and in this json format

"[
  {
    "bNo": 0,
    "cNo": 0,
    "vNo": 0,
    "vText": "King James Version"
  },"

Can anyone help pls?

**Comment by otseng** [comment]
We don't provide support for commercial or non-English Bibles.
